src/python/WMCore/JobStateMachine/ChangeState.py

```diff
@@ -89,6 +111,8 @@ def __init__(self, config, couchDbName=None):
         self.jsumdatabase = None
         self.statsumdatabase = None
 
+        # max total number of documents to be committed in the same Couch operation
+        self.maxBulkCommit = 250
```

Comment:
why this parameter is hard-coded? Can we put it into configuration?

Reply:
Updating.